sdk/servicebus/service-bus/test/utils/transitional.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Receiver } from "../../src/receivers/receiver";
+
+export function getDeadLetterPath(receiver: Receiver<{}>) {
```

Comment:
Update it to take the `entityPath` param so that we don't need to pass the receiever.

Reply:
I'll do one better....

Reply:
I've now pushed the getDeadLetterReceiver() change.